static/src/stylesheets/module/facia-garnett/_pillars.scss

```diff
@@ -415,7 +415,7 @@ $pillars: (
     background-color: $special-report-alt-faded;
 
     .fc-item__container.u-faux-block-link--hover {
-        background-color: darken($special-report-alt-faded, 2%);
+        background-color: darken($special-report-alt-pastel, 5%);
```

Comment:
### Hover state of feature

| Before      | After      |
|-------------|------------|
| ![image](https://user-images.githubusercontent.com/19683595/226954661-ea35e6c4-2203-46b3-a257-c06742bff761.png) | ![image](https://user-images.githubusercontent.com/19683595/226955674-0c2dbf26-fafb-4930-8bd2-5db73f9a0195.png) |
| ![image](https://user-images.githubusercontent.com/19683595/226954321-f9ad363b-48be-447b-be16-0269b3bf2100.png) | ![image](https://user-images.githubusercontent.com/19683595/226955823-7eb31be7-05ef-40b2-b104-5b24b2e33aab.png) |